geomesa-redis/geomesa-redis-tools/src/main/scala/org/locationtech/geomesa/redis/tools/RedisDataStoreCommand.scala

```diff
@@ -47,5 +48,8 @@ object RedisDataStoreCommand {
   trait RedisDataStoreParams extends CatalogParam {
     @Parameter(names = Array("--url", "-u"), description = "Redis connection URL", required = true)
     var url: String = _
+
+    @Parameter(names = Array("--cluster", "-cl"), description = "Redis cluster enabled?", required = false)
```

Comment:
I'm not sure it entirely matters, but we have standardized on single `-`'s with single chars, a la `getopts`. could you change the `-cl` to just be a single char flag (or omit it if it's more confusing since `-c` is already taken)? thanks!